jipp-core/src/main/java/com/hp/jipp/model/Types.kt

```diff
@@ -388,6 +414,11 @@ object Types {
      * [RFC8011](http://www.iana.org/go/rfc8011).
      */
     @JvmField val documentFormatDefault = StringType(Tag.mimeMediaType, "document-format-default")
+    /**
+     * "document-format-details" type as defined in:
+     * [PWG5100.7](https://ftp.pwg.org/pub/pwg/candidates/cs-ippjobext20-20190816-5100.7.pdf).
+     */
+    @JvmField val documentFormatDetails = AttributeCollection.Type("document-format-details", DocumentFormatDetails)
```

Comment:
if we are bringing this code back we need to remove this line 
```
ignored_attributes = [
    # XML Fix (obsolete or maybe just deprecated)
    'document-format-details-supported'
]
```

in that case the -supported will also come.